components/CardContent.js

```diff
@@ -10,10 +10,14 @@ const propTypes = {
   cardLink: PropTypes.node,
   cardInfo: PropTypes.array,
   className: PropTypes.string,
+  iconDescription: PropTypes.string,
 };
 
 const defaultProps = {
+  iconDescription: 'card icon',
   cardIcon: 'app-services',
+  cardTitle: 'card title',
+  cardInfo: ['info']
```

Comment:
Yeah, I wasn't sure about this one. Because you're right - I was just unsure about the scenario when the element _does_ render, each string in the `cardInfo` array needs to get translated. Will il8n still work if I omit `cardInfo` from `defaultProps`?